src/mumble/GlobalShortcut.ui

```diff
@@ -204,6 +204,26 @@
           </property>
          </spacer>
         </item>
+        <item>
+         <widget class="QPushButton" name="qpbImport">
+          <property name="toolTip">
+           <string>Import shortcuts from a file, replacing the current shortcuts.</string>
```

Comment:
Simplified English suggestion:
"Import shortcuts from a file. Overwrites the current shortcuts."